gdnative/gdnative.h

```diff
@@ -282,16 +286,16 @@ void GDAPI godot_print_error(const char *p_description, const char *p_function,
 void GDAPI godot_print_warning(const char *p_description, const char *p_function, const char *p_file, int p_line);
 void GDAPI godot_print(const godot_string *p_message);
 
-// GDNATIVE CORE 1.0.1
-
-bool GDAPI godot_is_instance_valid(const godot_object *p_object);
+// GDNATIVE CORE 1.0.2?
```

Comment:
Shouldn't it be 2.0.0, since there are breaking changes? e.g. PoolArray -> PackedArray